satellite/overlay/service.go

```diff
@@ -383,6 +396,12 @@ func (service *Service) UpdateUptime(ctx context.Context, nodeID storj.NodeID, i
 	return service.db.UpdateUptime(ctx, nodeID, isUp, lambda, weight, uptimeDQ)
 }
 
+// UpdateCheckIn updates a single storagenode's check-in info.
+func (service *Service) UpdateCheckIn(ctx context.Context, node NodeCheckInInfo) (err error) {
+	defer mon.Task()(&ctx)(&err)
```

Comment:
We need to resolve the IP address of the node here. See `Put`

Reply:
Alternatively, we could resolve the IP from satellite/contact endpoint and pass it in with NodeCheckInInfo

Reply:
see which `Put`? how do you resolve the IP from the endpoint?

Reply:
https://github.com/storj/storj/blob/54855ef8a247ec445e66fe218579ed7194cf1448/satellite/overlay/service.go#L317

Reply:
oh neat. great, let me add that here.